assets/embauche.html

```diff
@@ -34,18 +30,10 @@
 						&nbsp;€
 					</label>
 					<span class="input-help">Rémunération totale, dont primes. <br>Temps plein <em>(min. <span data-source="smic_proratise" data-round>1467</span>)</em> ou partiel. </span>
-
-					<select data-sets="#salaire@name">
-						<option value="salaire_de_base">brut</option>
-						<option disabled value="salaire_net_a_payer">net</option>
-					</select>
-
+					&nbsp;
+					<span>brut</span>
```

Comment:
The salary input hover won't be able to hide these texts if they're not inside spans.